plugin/pkg/admission/initialization/initialization.go

```diff
@@ -171,3 +294,71 @@ func copiedInitializers(names []string) *metav1.Initializers {
 		Pending: init,
 	}
 }
+
+// findInitializers returns the list of initializer names that apply to a config. It returns an empty list
+// if no initializers apply.
+func findInitializers(initializers *v1alpha1.InitializerConfiguration, gvr schema.GroupVersionResource) []string {
```

Comment:
In a followup we should reconcile this with the rule matcher the webhook system uses -- I like that one better ;)